pkg/util/containers/cri.go

```diff
@@ -0,0 +1,110 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2018 Datadog, Inc.
+
+// +build linux
```

Comment:
I don't know how many transitive dependencies `k8s.io/kubernetes/pkg/kubelet/apis/cri/runtime/v1alpha2` has, but to be safe, should we add a build flag to exclude it out of the puppy agent?